File: expressions/budget.c
#include <stdio.h>

int main(void){
    float income, rent, utilities, groceries, transportation, expenses, savings, total;
    float prent, putilities, pgroceries, ptranspotation, pexpenses;
    printf("This code is going to calculate your bugdet for the month. ");
    printf("How much do you make in a month?:\n");
    scanf("%f", &income);
    printf("How much do you pay in rent per month?:\n");
    scanf("%f", &rent);
    printf("How much do you pay in utilities per month?:\n");
    scanf("%f", &utilities);
    printf("How much do you pay in groceries per month?:\n");
    scanf("%f", &groceries);
    printf("How much do you pay in transportation in a month?:\n");
    scanf("%f", &transportation);
    expenses = rent + utilities + groceries + transportation;
    savings = income*.2;
    total = income-expenses-savings;
    prent = rent/income;
    putilities = utilities/income;
    pgroceries = groceries/income;
    ptranspotation = transportation/income;
    pexpenses = expenses/income;
    printf("Your income is: $%.2f\n", income);
    printf("Your expenses are: $%.2f\n", expenses);
    printf("Your savings are: $%.2f\n", savings);
    printf("Your total left to spend is: $%.2f\n", total);
    printf("Your rent payment is: %.2f", prent, "of your expenses");
    printf("Your utilities payment is: %.2f", putilities, "of your expenses");
    printf("Your groceries payment is: %.2f", pgroceries, "of your expenses");
    printf("Your transportation payment is: %.2f", ptranspotation, "of your expenses");
    return 0;
}

File: expressions/budget.py
print("This code will calculate your bugdet for the month.")
income = input("How much do you make in a month?\n")
Iincome = int(income)
rent = input("How much do you pay rent in a month?\n")
Irent = int(rent)
utilities = input("How much do you pay in utilities in a month?\n")
Iutil = int(utilities)
groceries = input("How much do you pay in groceries in a month?\n")
Igroc = int(groceries)
transportation = input("How much do you pay in transportation in a month?\n")
Itrans = int(transportation)
expenses = Irent + Iutil + Igroc + Itrans
Iexp = int(expenses)
savings = Iincome *0.2
Fsav = float(savings)
total = Iincome-Iexp+Fsav
prent = Irent/Iincome
putilities = Iutil/Iincome
pgroceries = Igroc/Iincome
ptranspotation = Itrans/Iincome
pexpenses = Iexp/Iincome
print("Your income is: $", income)
print("Your expenses are: $", expenses)
print("Your savings are: $", savings)
print("Your total left to spend is: $", total)
print("Your rent payment is: ", prent, "percent of your expenses")
print("Your utilities payment is: ", putilities, " percent of your expenses")
print("Your groceries payment is: ", pgroceries, " percent of your expenses")
print("Your transportation payment is: ", ptranspotation, " percent of your expenses")